### PR TITLE
Add loading spinner and change a bit of layout

### PR DESCRIPTION
**Pull request by iAmMrinal0**

### CODE REVIEW

File: kronor_swish/src/main/java/io/kronor/component/swish/SwishComponent.kt [hunk 1]
```diff
@@ -317,31 +322,34 @@ fun swishAppExists(context: Context): Boolean {
 
 @Composable
 fun SwishPromptMethods(onAppOpen: () -> Unit, onQrCode: () -> Unit, onPhoneNumber: () -> Unit) {
-    val doesSwishAppExist: Boolean = if (LocalInspectionMode.current) {
-        true
-    } else {
-        swishAppExists(LocalContext.current)
-    }
+    val doesSwishAppExist: Boolean =
+        LocalInspectionMode.current.takeIf { true } ?: swishAppExists(LocalContext.current)
```
Comment (pranaysashank) [inline kronor_swish/src/main/java/io/kronor/component/swish/SwishComponent.kt:326]: `this.takeIf p = if (p(this)) then this else null`

but what we want is

`this.takeIf p = if (this) then p else null`

So please revert this change

Comment (iAmMrinal0) [reply kronor_swish/src/main/java/io/kronor/component/swish/SwishComponent.kt]: not sure why that happened. I have changed it back to the old version for now.

File: kronor_swish/src/main/java/io/kronor/component/swish/SwishComponent.kt [hunk 2]
```diff
@@ -290,6 +290,7 @@ fun SwishPaymentWithPhoneNumber(onCancelPayment: () -> Unit) {
         Text(
             text = stringResource(R.string.accept_swish_phone), textAlign = TextAlign.Center
         )
+        CircularProgressIndicator()
```
Comment (pranaysashank) [inline kronor_swish/src/main/java/io/kronor/component/swish/SwishComponent.kt:293]: Let's remove this for now and merge it onto master

**Review by pranaysashank — APPROVED** [review]
